.gitignore

```diff
@@ -7,7 +7,10 @@
 /prt
 /output
 /examples/output
+/**/__pycache__
+/test-reports/
```

Comment:
Where does this folder come from?

Reply:
i've activated test report generation (junit xml), that's the location of the generated files...